### PR TITLE
Bugfix in FON

### PR DESCRIPTION
**Pull request by mcwimm**
so far the FON was not calculated if there was no tree-to-tree competition.
& added references for some equations

### CODE REVIEW

File: ResourceLib/BelowGround/Individual/FON/FON.py [hunk 1]
```diff
@@ -50,42 +50,52 @@ def calculateBelowgroundResources(self):
...
         """
+        self._r_stem = np.array(self._r_stem)
         distance = (((self._my_grid[0][:, :, np.newaxis] -
                       np.array(self._xe)[np.newaxis, np.newaxis, :])**2 +
                      (self._my_grid[1][:, :, np.newaxis] -
                       np.array(self._ye)[np.newaxis, np.newaxis, :])**2)**0.5)
-        my_fon = self.calculateFonFromDistance(np.array(self._r_stem),
-                                               distance)
+        my_fon = self.calculateFonFromDistance(distance=distance)
+        #print(my_fon)
```
Comment (jvollhueter) [inline ResourceLib/BelowGround/Individual/FON/FON.py:59]:
```suggestion
```

**Review by jvollhueter — CHANGES_REQUESTED** [review]
One print test still in code, otherwise ready to merge.

File: ResourceLib/BelowGround/Individual/FON/FON.py (hunk 1 above)
Comment (mcwimm) [reply ResourceLib/BelowGround/Individual/FON/FON.py]: fixed.

**Review by jvollhueter — APPROVED** [review]
Now ready to merge

**Pull request merged**